src/ajax.js

```diff
@@ -333,7 +349,15 @@ jQuery.extend({
 			"text html": true,
 
 			// Evaluate text as a json expression
-			"text json": jQuery.parseJSON,
+			"text json": function(json) {
```

Comment:
And I'm not sure that I would actually include an async JSON converter in the default build–you probably have to opt in by defining your own converter for "text json" in your `$.ajax` options.